File: 3195-separate-black-and-white-balls/separate-black-and-white-balls.cpp
class Solution {
public:
    long long minimumSteps(string s) {
        long long swaps = 0;
        long long white = 0;
        for (int i = 0; i < s.length(); i++){
            if (s[i] == '0'){
                swaps += i - white;
                white += 1;
            }
        }

        return swaps;
    }
};

File: 3195-separate-black-and-white-balls/separate-black-and-white-balls.py
class Solution:
    def minimumSteps(self, s: str) -> int:
        # swaps = 0
        # white = 0
        # for i in range(len(s)):
        #     if s[i] == '0':
        #         swaps += i - white
        #         white += 1
        # return swaps                
        swaps = 0
        black = 0
        for i in s:
            if i == '1':
                black += 1
            else:
                swaps += black
        return swaps

